doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -859,6 +859,77 @@ However elimination to `Type` or to a polymorphic sort with `s := Prop` is allow
         : Type@{s|max(u,v)}
         := pair { pr1 : A; pr2 : B pr1 }.
 
+Explicit Sorts
+---------------
+
+Similarly to universes, fresh global sorts can be declared with the :cmd:`Sort`.
+
+.. cmd:: Sort {+ @ident }
+         Sorts {+ @ident }
+
+   In the monomorphic case, declares new global sort qualities 
+   with the given names.  Global quality names live in their own namespace.
+   Inside sections, the command respects the `Universe Polymorphism` flag,
+   either set globally or locally through the :attr:`universes(polymorphic)` attribute (or
+   the ``Polymorphic`` legacy attribute), meaning the sort
+   quantification will be discharged for each section definition
+   independently. Polymorphic sort qualities are forbidden outside sections.
+
+  .. exn:: Polymorphic sorts can only be declared inside sections, use #[universe(polymorphic=no)] Sort instead.
+
+    This error spwans when a :cmd:`Sort` command is issued outside a section with universe polymorphism set. 
```

Comment:
```suggestion
    Generated when a :cmd:`Sort` command is issued outside a section with universe polymorphism set. 
```

We try to avoid repetitive boilerplate such as "This error..."